modules/apps/frontend-css/frontend-css-web/src/main/resources/META-INF/resources/portal/_forms.scss

```diff
@@ -90,6 +90,10 @@
 	}
 }
 
+.input-group .form-validator-stack {
```

Comment:
Should this go in the [compat layer](https://github.com/liferay/liferay-portal/tree/master/modules/apps/frontend-theme/frontend-theme-styled/src/main/resources/META-INF/resources/_styled/css/compat/components) since we are working with old markup?